docker/mongodb-kubernetes-tests/tests/authentication/mongodb_custom_roles.py

```diff
@@ -0,0 +1,260 @@
+from kubetester import (
+    create_or_update_configmap,
+    find_fixture,
+    random_k8s_name,
+    read_configmap,
+    try_load,
+    wait_until,
+)
+from kubetester.mongodb import MongoDB, Phase
+from kubetester.mongodb_multi import MongoDBMulti
+from kubetester.mongodb_role import ClusterMongoDBRole, ClusterMongoDBRoleKind
+from pytest import fixture, mark
+from tests.multicluster.conftest import cluster_spec_list
+
+
+@fixture(scope="module")
+def project_name_prefix(namespace: str) -> str:
+    return random_k8s_name(f"{namespace}-project-")
+
+
+@fixture(scope="module")
+def first_project(namespace: str, project_name_prefix: str) -> str:
+    cm = read_configmap(namespace=namespace, name="my-project")
+    project_name = f"{project_name_prefix}-first"
+    return create_or_update_configmap(
```

Comment:
I did a similar thing but wrapped in .configure function in this (not yet merged PR): https://github.com/mongodb/mongodb-kubernetes/pull/173/files#diff-9bd4ee69baff1e418bdf57281d036ca298a0b4aa5628cfc457b41e6b0fe565e7R111

I've left a comment there to change it after this PR is merged, so nothing to be done here for you.